source/process/casestudy.rst

```diff
@@ -92,4 +92,16 @@ If you're completing the questionnaire by email, please send a copy of the quest
 
 Thank you so much for being part of our community!
 
+Written Case Study Lay Out
+----------------------------
+
+Follow these steps to write a case study that is clear and concise:
+
+1. Title: Focus on highlighting the most compelling accomplishment.
+2. Executive Summary: Begin the case study with a section that includes a powerful hook and a summary of the most important points of the entire story.
```

Comment:
Wondering if someone reading this in Japan understands the word `powerful hook`. 0/5